references/cicd-sharedflow-pipeline/pipeline.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -x
+
+TOKEN=$(gcloud auth print-access-token)
+
+# Deploy the sharedflow
+mvn clean install -ntp -B -Pgoogleapi -Dorg="$APIGEE_X_ORG" -Denv="$APIGEE_X_ENV" \
```

Comment:
`-ntp` is only supported in 3.6.1 onwards. The default maven version in debian is 3.6.0, so for the best compatibility we shouldn't use this flag.

please consider replacing `-ntp` with a different option or specifying 3.6.1 as the minimum maven version in the README

Reply:
Good catch. Had it removed everywhere except that. Fixed it now